.github/workflows/maven.yml

```diff
@@ -0,0 +1,29 @@
+name: Java Release to Sonatype
+
+on:
+ workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Java for publishing to Maven Central Repository
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          server-id: ossrh
+          server-username: MVN_USER 
```

Comment:
Are you sure this should not read: `server-username: ${{ secrets.MVN_USER }}`

Reply:
sorry, may bad, you are right. I didn't quite get how the variables worked, it seems.